src/gui/AstroCalcDialog.cpp

```diff
@@ -3749,6 +3756,311 @@ void AstroCalcDialog::saveSolarEclipsesLocal()
 	}
 }
 
+void AstroCalcDialog::setTransitHeaderNames()
+{
+	transitHeader.clear();
+	transitHeader << q_("Date");
+	// TRANSLATORS: The name of column in AstroCalc/Transits tool
+	transitHeader << qc_("Planet", "column name");
+	// TRANSLATORS: The name of column in AstroCalc/Transits tool
+	transitHeader << qc_("Separation", "column name");
```

Comment:
Why the column is named 'separation'? What is mean 'separation' here?

Reply:
It's minimum angular distance between the planet and Sun's center. I put that into tooltip (we can change it to "Angular distance" or something like that for clearer meaning).

Reply:
OK, I see your explanation and I found related part in the code :) Thanks! I think we should use clearer term here. 

I think the comment `// TRANSLATORS: The name of column in AstroCalc/Eclipses/Transits tool` will be better also.

Reply:
I think we can really leave away "column name" in the translation string of "Planet", thus re-using/saving one translation item. 
The term "separation" is OK for me, esp. when explained in the tooltip. But distance is also OK.

Reply:
> I think we can really leave away "column name" in the translation string of "Planet", thus re-using/saving one translation item.

We don't have translatable string exactly "Planet" :)

Reply:
Uh. Yes, we have e.g. "planet", "Planet:", "Planets", and tens of combinations. But it's such a general term that we don't have to self-limit its use as "column name". it may easily be used elsewhere.

Reply:
I guess changing it to q_("Planet")